app/src/main/res/layout/mpower2_study_burst_status_bar.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                                             xmlns:app="http://schemas.android.com/apk/res-auto"
+                                             xmlns:tools="http://schemas.android.com/tools"
+                                             android:layout_width="match_parent"
+                                             android:layout_height="match_parent">
+
+    <android.support.constraint.ConstraintLayout
+        android:id="@+id/title_text_background"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_marginBottom="5dp"
+        android:layout_marginTop="5dp"
+        android:background="@drawable/study_burst_title_text_background"
+        app:layout_constraintBottom_toBottomOf="@+id/study_burst_progress_bar_dial"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/study_burst_progress_bar_dial"
+        app:layout_constraintTop_toTopOf="@+id/study_burst_progress_bar_dial">
+
+        <TextView
+            android:id="@+id/study_burst_progress_bar_title"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="12dp"
+            android:fontFamily="@font/lato"
+            android:text="Study Burst"
+            android:textColor="@color/appWhite"
+            android:textSize="18sp"
```

Comment:
Let's have styles that define font family, size, color. Or at least constants for the sizes

Reply:
I'm also going to do this in another pr.